docs/en/custom-elements/supplemental.md

```diff
@@ -0,0 +1,143 @@
+# Custom Element Events
+
+Function based properties that are prefixed with "on" are automatically converted to events when building a widget as a custom element. This allows you to use the standard browser event listeners to handle those properties.
+
+In addition to being added as events, event properties are also exposed as regular properties on the custom element. This allows you to assign a function callback directly, without using an event handler.
+
+```html
+<script>
+    emailSignup.__onSignUp = () => {
```

Comment:
hmm there is a reason these are double underscored, these are magic properties for parent dojo widgets use and not designed ever to be used by the end user.

Reply:
Sounds good! I'll remove this section.